integrations/disco/routes/post-credential-issue/client.ts

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios'
+
+import { IssuedCredentials } from '../../utils/types'
+
+export async function appDiscoPostCredentialIssue(values: { recipientDid: any }) {
+  try {
+    const payload = {
+      schemaUrl: 'https://raw.githubusercontent.com/discoxyz/disco-schemas/main/json/ProofOfHackCredential/1-0-0.json',
+      subjectData: { ...values },
+      recipientDID: values?.recipientDid,
+    }
+    const { data }: { data: IssuedCredentials } = await axios.post(`/api/disco/credential-issue`, JSON.stringify(payload), {
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    })
+    return data
+  } catch (error) {
+    throw error
+  }
```

Comment:
axios automatically handles errors, there's no need for a `try...catch` here

Reply:
removing the same